### ISSUE TITLE
Do we need Windows special LIBRARY and INC appends?

### ISSUE THREAD

**Issue by wolfv** [prefix-dev/rattler-build#1107]
conda-build has the following code:

```py
fo.write('set "INCLUDE={};%INCLUDE%"\n'.format(env["LIBRARY_INC"]))
fo.write('set "LIB={};%LIB%"\n'.format(env["LIBRARY_LIB"]))
```

Things seem to work OK without but I guess we should still add these variables.

In an old xtensor build I can observe:

```
2024-01-25T22:30:24.7610181Z (base) %SRC_DIR%>set "INCLUDE=%PREFIX%\Library\include;" 
2024-01-25T22:30:24.7615463Z 
2024-01-25T22:30:24.7616111Z (base) %SRC_DIR%>set "LIB=%PREFIX%\Library\lib;" 
```

**Comment by wolfv** [comment]
> @isuruf wrote on Element: `LIB` and `INCLUDE` are used by MSBuild. Also some projects use it to find the SDK directories and VS directories as these env variables are appended to by `vsvarsall.bat`.

I think we should add it but ideally we add a function to rattler to "append" variables so that we can do it cleanly.

**Comment by wolfv** [comment]
Fixed by #1118
